iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -1030,6 +1039,48 @@ static void copyFromAliasingBufferToResultBuffer(OpBuilder &b, Location loc,
   }
 }
 
+static LogicalResult convertPadTensorOp(
+    OpBuilder &b, linalg::PadTensorOp padTensorOp, BlockAndValueMapping &bvm,
+    BufferizationPlan &plan, WorkgroupMemoryAllocationFn allocationFn) {
+  auto inputTensor = padTensorOp.source();
+  auto inputMemref = bvm.lookup(inputTensor);
+
+  auto loc = padTensorOp.getLoc();
+
+  if (failed(getOrAllocateResultBuffers(b, padTensorOp, padTensorOp.result(),
+                                        bvm, plan, allocationFn))) {
+    return failure();
+  }
+
+  auto resultPaddedBuffer = bvm.lookup(padTensorOp.result());
+
+  // Get padding value and fill the result buffer.
+  linalg::YieldOp yeildOp =
```

Comment:
This is assuming that the `yield` has a value computed from `std.constant`. Check for that and abort if that is not the case.

Reply:
Shouldn't that be verified by the `PadTensorOP::verifier` ?

Reply:
I dont think it is. `pad_tensor` is meant to be more general (if it were always a constant, there would be no reason to have a region and `yield` the value.)